lib/portlayer/storage/data_source.go

```diff
@@ -53,6 +54,46 @@ func (m *MountDataSource) Source() interface{} {
 }
 
 // Export reads data from the associated data source and returns it as a tar archive
+func (m *MountDataSource) Stat(op trace.Operation, spec *archive.FilterSpec) (*FileStat, error) {
```

Comment:
If we're passing in a FilterSpec as the input, we should return a `map[string]*FileStat` for the result - while it's fine for an implementation to return an error stating that it cannot deal with multiple inputs the signature should be consistent between intput/output to allow for other implementation decisions.